GPy/core/parameterization/priors.py

```diff
@@ -357,24 +357,17 @@ class InverseGamma(Gamma):
     """
     domain = _POSITIVE
     _instances = []
-    def __new__(cls, a=1, b=.5): # Singleton:
```

Comment:
Does removing this mean that those default settings are lost? Or are they the same as the original Gamma?

Reply:
They're identical to the defaults from `class Gamma(Prior)`'s `__new__` as defined on `GPy/core/parameterization/priors.py#L291` in `devel:HEAD` at time of writing (i.e. at commit `ff82f12c3d321bfc3ce6615447fad25aea9de6bd`).